hs_file_types/models/base.py

```diff
@@ -772,6 +762,10 @@ def get_main_file(self):
                         return f
         return None
 
+    @property
+    def url(self):
+        return "/" + os.path.join("resource", self.resource.file_path, self.aggregation_name)
```

Comment:
Remove `+` by moving `/` into os.path.join call, i.e., os.path.join('/', 'resource', ...)

Reply:
cool, thanks for that. updated.